### DISCUSSION TITLE
How to get   mask_loss

### DISCUSSION THREAD

**Discussion by donghang941114** [STVIR/pysot#255] [General] [answered]
if cfg.MASK.MASK:
            # TODO
            mask, self.mask_corr_feature = self.mask_head(zf, xf)
            mask_loss = None
            outputs['total_loss'] += cfg.TRAIN.MASK_WEIGHT * mask_loss
            outputs['mask_loss'] = mask_loss
        return outputs

**Comment by ZhiyuanChen** [comment, marked as answer]
The TODO flag means it hasn't been implemented yet.
